src/odemis/acq/stream/_sync.py

```diff
@@ -241,7 +248,7 @@ def estimateAcquisitionTime(self):
         for l in self.leeches:
             shape = (len(pol_pos), rep[1], rep[0])
             # estimate acq time for leeches is based on two fastest axis
-            if hasattr(self._sccd, "integrationTime"):
+            if self._integrationTime:
```

Comment:
Let's update every place with such code to use `._integrationTime` and `._integrationCounts`. This ensure that we don't get confused in the long term with some code using the wrong variable.